src/main/java/org/glyptodon/guacamole/auth/json/connection/ConnectionService.java

```diff
@@ -205,15 +206,18 @@ public GuacamoleTunnel connect(UserData.Connection connection,
         String hostname = proxyConfig.getHostname();
         int port = proxyConfig.getPort();
 
-        // Generate and verify connection configuration
```

Comment:
Mind restoring this comment? Feels dirty to be _removing_ documentation. ;)

Reply:
@mike-jumper for sure. Added it back